content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -431,444 +431,9 @@ Some values of an object are typically generated before the object is persisted.
-`kubectl apply`.
-
-This behavior applies to server-side apply with the `kubectl` field manager. As
-an exception, you can opt-out of this behavior by specifying a different,
-non-default field manager, as seen in the following example. The default field manager for kubectl
-server-side apply is `kubectl`.
-
-```shell
-kubectl apply --server-side --field-manager=my-manager [--dry-run=server]
-```
-
-### API Endpoint
-
-With the Server Side Apply feature enabled, the `PATCH` endpoint accepts the
-additional `application/apply-patch+yaml` content type. Users of Server Side
-Apply can send partially specified objects as YAML to this endpoint.
-When applying a configuration, one should always include all the fields
-that they have an opinion about.
-
-### Clearing ManagedFields
-
-It is possible to strip all managedFields from an object by overwriting them
-using `MergePatch`, `StrategicMergePatch`, `JSONPatch` or `Update`, so every
-non-apply operation. This can be done by overwriting the managedFields field
-with an empty entry. Two examples are:
-
-```console
-PATCH /api/v1/namespaces/default/configmaps/example-cm
-Content-Type: application/merge-patch+json
-Accept: application/json
-Data: {"metadata":{"managedFields": [{}]}}
-```
-
-```console
-PATCH /api/v1/namespaces/default/configmaps/example-cm
-Content-Type: application/json-patch+json
-Accept: application/json
-Data: [{"op": "replace", "path": "/metadata/managedFields", "value": [{}]}]
-```
-
-This will overwrite the managedFields with a list containing a single empty
-entry that then results in the managedFields being stripped entirely from the
-object. Note that just setting the managedFields to an empty list will not reset
-the field. This is on purpose, so managedFields never get stripped by clients
-not aware of the field.
-
-In cases where the reset operation is combined with changes to other fields than
-the managedFields, this will result in the managedFields being reset first and
-the other changes being processed afterwards. As a result the applier takes
-ownership of any fields updated in the same request.
-
-{{< caution >}}
-Server Side Apply does not correctly track ownership on
-sub-resources that don't receive the resource object type. If you are
-using Server Side Apply with such a sub-resource, the changed fields
-won't be tracked.
-{{< /caution >}}
-
-### Disabling the feature
-
-Server Side Apply is a beta feature, so it is enabled by default. To turn this
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates) off,
-you need to include the `--feature-gates ServerSideApply=false` flag when
-starting `kube-apiserver`. If you have multiple `kube-apiserver` replicas, all
-should have the same flag setting.
+Starting from Kubernetes v1.18, you can enable the
+[Server Side Apply](/docs/reference/using-api/server-side-apply/)
+feature so that the control plane tracks managed fields for all newly created objects.
```

Comment:
Here is another link to the new page.

Reply:
Last point. I don't think you should duplicate the feature gate information here. The version/feature information
is found in the SSA page.

This section should indicate to the reader that this content is found in another section.
I suggest to introduce the SSA concept (very briefly, 1-2 sentences)  then link to the SSA page.
I would not duplicate the content. Something like:

_Server Side Apply_ helps users and controllers to manage their resources through
declarative configurations. To learn about resource field management, see [Server Side Apply](/docs/reference/using-api/server-side-apply/).

Reply:
Well ... the doc says "you can enable the Server Side Apply feature", not "the Server Side Apply feature gate".
I did tried to duplicate more meaningful info into this short section, however, the following paragraph is not a good overview.

> Server Side Apply helps users and controllers to manage their resources through
declarative configurations. To learn about resource field management, see Server Side Apply.

Reply:
OK. If you want to introduce SSA as proposed, then I suggest removing
lines 17-18 from the new page or edit the content so there is no duplication.
I'd like one sentence that defines SSA (without telling me how to enable it).
Providing hints about "what's next" is helpful to readers.
I might move the link to the SSA section to the last sentence.
Here is a suggestion:

Starting from Kubernetes v1.18, you can enable the
[Server Side Apply](/docs/reference/using-api/server-side-apply/)
feature so that the control plane tracks managed fields for all newly created objects.
Server Side Apply introduces [or provides] a clear pattern for managing field conflicts,
offers server-side `Apply` and `Update` operations, and replaces the client-side functionality 
of `kubectl apply`. For more details about these features [or about managed fields], see the section on [Server Side Apply](...).

Reply:
Done.

Reply:
I've revised the text as suggested, but forgot to `git push -f`. Sorry.